packages/falso/src/tests/social.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { randSocial } from '../lib/social';
+
+describe('Social', () => {
+  it('Shuould return social object in formats [{name, link}]', () => {
+    expect(randSocial).toBeTruthy();
+  });
```

Comment:
The test doesn't reflect the "should" statement

Reply:
I think now the test should be ok.
https://github.com/oyuutan/falso/commit/88543249affce89a7b1cf88529b19d8cdf34475c